src/app/features/courses/courses.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.css']
```

Comment:
.scss file when declaring the stylesUrls

Reply:
When I use CLI to generate components I was careless and unfortunately leave those extentions like css.